.github/workflows/test-and-build.yml

```diff
@@ -30,3 +30,6 @@ jobs:
 
     - name: Test backend
       run: yarn test
+
+    - name: Test running scheduler
+      run: yarn workspace backend scheduler
```

Comment:
Simple test to check that it runs and exists successfully.. (we had to check CI flag)